components/BottomSheet/src/MDCBottomSheetPresentationController.m

```diff
@@ -285,6 +288,13 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
   }
 }
 
+- (void)setDismissOnDraggingDownSheet:(BOOL)dismissOnDraggingDownSheet {
+  _dismissOnDraggingDownSheet = dismissOnDraggingDownSheet;
+  if (self.sheetView) {
```

Comment:
When we load the `sheetView` should we set this property on it?

Reply:
yep, this is already done